test/WebJobs.Script.Tests/NodeEndToEndTests.cs

```diff
@@ -224,6 +224,7 @@ public void ErrorFunction_DoesNotBringDownHost()
             JObject resultObject = JObject.Parse(body);
             Assert.Equal((string)resultObject["reqBodyType"], "object");
             Assert.Equal((string)resultObject["reqBody"]["testData"], testData);
+            Assert.Equal((string)resultObject["bindingData"]["testData"], testData);
```

Comment:
Verifying that the binding data was populated is sufficient verification. I also did some sanity e2e tests - verified a blob binding worked (the original bug repro)